### PR TITLE
Update version matrix in readme

### PR DESCRIPTION
**Pull request by jdreesen**
I just guessed the `Status` column. Please let me know if this is correct.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,10 +347,11 @@ container:
...
-| Version | Released     | PHPUnit     | Status   |
-|---------|--------------|-------------|----------|
-| 5.x     | Jan 23, 2024 | 9.x, 10.x   | Latest   |
-| 4.x     | Mar 28, 2019 | 8.x and 9.x | Bugfixes |
-| 3.x     | Mar 5, 2018  | 7.x         | Bugfixes |
-| 2.x     | May 9, 2017  | 6.x         | Bugfixes |
-| 1.x     | Jul 4, 2016  | 4.x and 5.x | EOL      |
+| Version | Released     | PHPUnit       | Status     |
+|---------|--------------|---------------|------------|
+| master  | Jan 23, 2024 | 10.5 and 11.x | Latest     |
```
Comment (jdreesen) [inline README.md:352]: The supported versions changed in `master` (via #157). 

Should we leave this out until it's released?

**Review by jdreesen — COMMENTED** [review]
Fixed.

File: README.md (hunk 1 above)
Comment (Nyholm) [reply README.md:352]: Let's add 6.x.

**Comment by mbabker** [comment]
Thanks!

**Pull request merged**